src/modules/mount/main.py

```diff
@@ -307,6 +307,12 @@ def mount_partition(root_mount_point, partition, partitions, mount_options, moun
                                         mount_option) != 0:
                 libcalamares.utils.warning("Cannot mount {}".format(device))
 
+def enable_swap_partition(swap_partitions):
```

Comment:
This should probably be renamed to be plural now that it supports more than one.